src/MorganStanley.ComposeUI.MainApp/App.axaml.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Avalonia;
```

Comment:
Source files in this project are missing license headers

Reply:
I'll check them again

Reply:
fixed